apps/ledger-live-mobile/src/components/Carousel/defaultSlides.ts

```diff
@@ -0,0 +1,14 @@
+import { SlideName } from "./types";
+
+export const defaultSlides: SlideName[] = [
+  SlideName.ManageClCard,
+  SlideName.Swap,
+  SlideName.Lido,
+  SlideName.buyCrypto,
+  SlideName.Market,
+  SlideName.FamilyPack,
+];
+
+export const defaultSlidesVisibilityState = Object.fromEntries(
+  defaultSlides.map(slideName => [slideName, true]),
+);
```

Comment:
all changes in the `Carousel` directory are unrelated to the general goal of this PR but there was a require cycle here that was itching me 🧹